business-wiki-frontend/shared/components/SidepanelListElement.tsx

```diff
@@ -0,0 +1,13 @@
+import moduleStyles from "../styles/SidepanelListElement.module.scss";
+
+import { SidepanelListElementInterface } from "../models/SidepanelListElementInterface";
+
+const SidepanelListElement = (props: { value: SidepanelListElementInterface }) => {
```

Comment:
Destrukturyzacja propsów.

Reply:
Resolved in 22cdae1140ab41725990d1016865c3098c76b73a